src/components/ListOrgs.vue

```diff
@@ -25,6 +25,11 @@
               <Chip v-for="(tag, index) in node.data.tags" :label="tag" :key="index" icon="pi pi-tag" class="m-1" />
             </template>
           </Column>
+          <Column field="" header="" #body="{ node }">
+            <router-link :to="{ name: 'ListUsers', params: { orgType: node.data.orgType, orgId: node.data.id } }">
+              <Button label="View users" />
+            </router-link>
```

Comment:
For consistency with the buttons in the AdministrationProgress PR:

```suggestion
            <router-link :to="{ name: 'ListUsers', params: { orgType: node.data.orgType, orgId: node.data.id } }">
              <Button v-tooltip.top="'View users'" severity="secondary" text raised label="Users" aria-label="View users" icon="pi pi-users" size="small" />
            </router-link>
```

Or for a more minimalist look that I prefer but that might not fly with all of the partnerships folks
```suggestion
            <router-link :to="{ name: 'ListUsers', params: { orgType: node.data.orgType, orgId: node.data.id } }">
              <Button v-tooltip.top="'View users'" severity="secondary" outlined aria-label="View users" icon="pi pi-users" />
            </router-link>
```

Might have to play with the `size` attribute on that second suggestion.

Reply:
Implemented the consistent style. To be fair though, I think I like the icon design as well.